### DISCUSSION TITLE
How can I swap floppy image or iso image on the fly?

### DISCUSSION THREAD

**Discussion by QianNangong** [utmapp/UTM#3089] [UTM Help (macOS)]
I'd like to install some old OSs which installation media is on multiple floppies or cds...
The removable devices menu is grayed when vm is running...

**Comment by melvinlusk** [comment]
Did you figure this out? I have the same issue.

**Comment by melvinlusk** [comment]
I figured it out.  You have select the floppy image as "Removable".

**Comment by steveparry1** [comment]
I have an external USB floppy disc drive which I can't access on my OS9.2 VM. I know it works because I've seen it mount discs in a Linux VM. But it won't show up in Mac OS 9.2 either with a disc in or without a disc. Can anyone help?
